lib/generators/tailwindcss/mailer/mailer_generator.rb

```diff
@@ -3,7 +3,7 @@
 module Tailwindcss
   module Generators
     class MailerGenerator < Erb::Generators::MailerGenerator
-      source_root File.expand_path("../templates", __FILE__)
+      source_root File.expand_path("templates", __dir__)
```

Comment:
Same comment here, this will not work if someone want to override mailer templates

Reply:
Fixed